test/e2e/storage/testsuites/volumes.go

```diff
@@ -118,12 +118,11 @@ func (t *volumesTestSuite) defineTests(driver TestDriver, pattern testpatterns.T
 
 		// Now do the more expensive test initialization.
 		l.config, l.testCleanup = driver.PrepareTest(f)
+		l.intreeOps, l.migratedOps = getMigrationVolumeOpCounts(f.ClientSet, dInfo.InTreePluginName)
```

Comment:
why are we moving the metrics getting only for the `volumes.go` test suites? (this is being done in each of the test suits)

Reply:
Every other call is in the right place for some reason, i.e. after PrepareTest and before resource creation

The other reference to `getMigrationVolumeOpCounts` is inside a function that's always called in cleanup, after resource deletion